oak/proto/permissions.proto

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.permissions;
+
+// A PermissionsConfiguration provides a declarative description of the features that are permitted
+// for an Oak application.
```

Comment:
Didn't we arrive to the conclusion that this should be a TOML file instead of proto in the end? Otherwise, how is it going to be parsed? Or did you figure out if / how it can be done in Rust using textproto format?

Reply:
I might be missing something here, but I think it can be a protobuf, the same way we have `application.proto`, but the actual permission files that we use as input to the `oak_loader` will be toml files, the same way we now have a `config.toml` for each example. Alternatively, this could be a simple `struct`, but I thought having it as a protobuf is more consistent with the rest of the codebase.

Reply:
since we need to define the actual struct for TOML deserialization (something like [this](https://github.com/project-oak/oak/blob/bfc2a023dba2611e115f647d79369f0e31ceaf80/runner/src/main.rs#L582-L589)), I don't see how the protobuf helps with anything?

Reply:
Replaced the protobuf with a struct.